gst/nnstreamer/nnstreamer_conf.c

```diff
@@ -345,3 +353,35 @@ nnsconf_get_fullpath (const gchar * subpluginname, nnsconf_type type)
   g_free (filename);
   return ret;
 }
+
+/** @brief Public function defined in the header */
+const gchar *
+nnsconf_get_value (nnsconf_type_value type)
+{
+  int i;
+  gchar *ret;
+
+  switch (type) {
+    case NNSCONF_VAL_TF_MEM_OPTMZ:
+      for (i = 0; i < CONF_SOURCES; i++) {
+        if (conf.valueTF_MEM_OPTMZ[i]) {
+          conf.retTF_MEM_OPTMZ = g_malloc_n (1, sizeof (gchar *));
+          if (!g_ascii_strncasecmp ("1", conf.valueTF_MEM_OPTMZ[i], 1) ||
+              !g_ascii_strncasecmp ("t", conf.valueTF_MEM_OPTMZ[i], 1))
+            conf.retTF_MEM_OPTMZ = "1";
```

Comment:
You are writing conf.retTF_MEM_OPTMZ[1]='\0', which is array out-of-bound.
Rewrite:
```
conf.retTF_MEM_OPTMZ[0] = '1';
```

Reply:
it's updated

Reply:
because of the below error, I made the size as 2
```bash
error: assignment makes pointer from integer without a cast [-Werror=int-conversion]
             conf.retTF_MEM_OPTMZ = '1';
```

Reply:
> error: assignment makes pointer from integer without a cast [-Werror=int-conversion]
             conf.retTF_MEM_OPTMZ = '1';

It is because you are missing the index. :)  (```[0]``` is missing)